Clients/GUI/Pages/InstrumentInfo.razor

```diff
@@ -0,0 +1,469 @@
+        </div>
+    </div>
+    <div class="col-md-6">
+        <h3 align="center">@instrument.Name</h3>
+        <br>
+        <div class="container">
+            @if (isInstrumentLoading)
+            {
+                <div class="spinner-border" role="status">
+                    <span class="sr-only">Loading...</span>
+                </div>
+            }
+            else if (!errorOccured)
+            {
+                <div>
+                    <div class="form-row input-group-prepend">
+                        <label class="col-3">Current balance </label>
+                        <input type="text" disabled="disabled" class="col-3" @bind-value=@currency>
+                        <input type="number" disabled="disabled" class="col-3" @bind-value=@currentBalance>
+                    </div>
+                    <br>
+                    <div class="form-row input-group-prepend">
+                        <label class="col-3">Update balance</label>
+                        <input type="number" class="col-3" @bind-value=@cash>
+                        <button class="btn btn-primary col-3" value="Update" @onclick="@UpdateBalance">Update</button>
+                    </div>
+                </div>
+                <br>
+
+                <table class="table table-bordered">
+                    <thead class="thead-light">
+                        <tr>
+                            <th scope="col">Figi</th>
+                            <th scope="col">Name</th>
+                            <th scope="col">Type</th>
+                            <th scope="col">Currency</th>
+                            <th scope="col">Count in lot</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>@instrument.Figi</td>
+                            <td>@instrument.Name</td>
+                            <td>@instrument.Type</td>
+                            <td>@instrument.Currency</td>
+                            <td>@instrument.CountInLot</td>
+                        </tr>
+                    </tbody>
+                </table>
+
+                <div class="form-row">
+                    <div class="form-group col-md-3">
+                        <label for="countInPortfolio">Count in portfolio</label>
+                        <input type="number" disabled="disabled" class="form-control" id="countInPortfolio" @bind-value=@countInPortfolio>
+                    </div>
+                    <div class="form-group col-md-2">
+                        <label for="lotsToTrade">Lots to trade</label>
+                        <input type="number" class="form-control" id="lotsToTrade" @bind-value=@lotsToTrade>
+                    </div>
+                    <div class="form-group col-md-2">
+                        <label for="price">Price</label>
+                        <input type="number" class="form-control" id="price" @bind-value=@price>
```

Comment:
What is this? Why we can change price?

Reply:
Because now we send a request with the price that we choose. We can set any price in the sandbox and deal would be done